docs/devguide/newdashboards.asciidoc

```diff
@@ -63,16 +67,18 @@ This command does initial setup of the environment:
  * Index mapping template in Elasticsearch to ensure fields are mapped.
  * Kibana dashboards (where available).
  * ML jobs (where available).
+ * Ingest pipelines (where available).
+ * ILM policy (for Elasticsearch 6.5 and newer).
```

Comment:
@simitt Note that this the exact output of running `metricbeat help setup` (mentions 6.5), so I don't think I should change the version here.

Reply:
Thanks @dedemorton, afaik it was _possible_ to import the policies from 6.5 on (beta), but one would need to set `setup.ilm.enabled: true` as with `auto` it only enables and therefore loads policies for ES >= `7.0`. I think it is ok to stick with this text, as it doesn't specify any further.